pkg/kubelet/kubelet.go

```diff
@@ -496,9 +506,18 @@ func NewMainKubelet(kubeCfg *componentconfig.KubeletConfiguration, kubeDeps *Kub
 	case "docker":
 		switch kubeCfg.ExperimentalRuntimeIntegrationType {
 		case "cri":
+			networkPlugin, err := network.PickNetworkPlugin(kubeDeps.NetworkPlugins, kubeCfg.NetworkPluginName)
```

Comment:
I feel like the whole network plugin probing and picking part should be in the server wrapper, which does not exist today. Hence I leave this part out here.